docs/includes/usage-examples/CountTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Models\Movie;
+use MongoDB\Laravel\Tests\TestCase;
+
+class CountTest extends TestCase
+{
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testCount(): void
+    {
+        require_once __DIR__ . '/Movie.php';
+
+        Movie::truncate();
+        Movie::insert([
+            [
+                'title' => 'Young Mr. Lincoln',
+                'genres' => ['Biography', 'Drama'],
+            ],
+            [
+                'title' => 'Million Dollar Mermaid',
+                'genres' => ['Biography', 'Drama', 'Musical'],
+            ],
+        ]);
+
+        // begin-count
+        $biographies = Movie::where('genres', 'Biography')
+            ->count();
```

Comment:
The 2nd example for "a collection of object" would be:

```php
$biographies = Movie::where('genres', 'Biography')->get();
$count = $biographies->count();
```

Reply:
Does the `where()` method return a collection of objects? If so, it seems (from my understanding) accurate to say that I'm calling `count()` on a "collection of objects" if I chain `count()` right after `where()`. If not, I'll change the other code examples that mention a "collection of objects" as well to use `get()`.

Reply:
`where()` returns a query builder. Calling `->get()` runs the find query and returns the collection of model instances.